conf/quiz/quiz-config.yml

```diff
@@ -71,6 +71,7 @@ rounds:
         question: Order these cities from small to large
         questionDetail: Population according to Google on July 2020
         masterNotes: Mention that this is not a trick question
+        image: https://static.thenounproject.com/png/3315776-200.png
```

Comment:
Does this image have the right copyright to be included here?

Reply:
You're wright, i've just checked, it's CreativeCommon and it need to be credited